internal/common/resource/resource.go

```diff
@@ -9,6 +9,11 @@ import (
 	"k8s.io/apimachinery/pkg/api/resource"
 )
 
+// FromResourceList function takes a map with keys of type ResourceName and values of type
+// "resource.Quantity" as defined in the K8s API.
+//
+//	It converts the keys to strings and creates a new map with the same keys, but with deep copies of the values.
```

Comment:
Can you keep spacing the same? 
```suggestion
// It converts the keys to strings and creates a new map with the same keys, but with deep copies of the values.
```